site24x7/websitemonitor.go

```diff
@@ -212,7 +215,7 @@ type WebsiteMonitor struct {
 	UserGroupIDs          []string         `json:"user_group_ids"`
 	ActionIDs             []ActionRef      `json:"action_ids,omitempty"`
 	UseNameServer         bool             `json:"use_name_server"`
-	NotifyToPagerDuty     bool             `json:"notify_to_pager_duty"`
+	ThirdPartyServices    []string         `json:"third_party_services"`
```

Comment:
I think this needs to be `json:"third_party_services,omitempty"` otherwise the Site24x7 api responds with a 400 malformed json error when you don't have any thirdparty services.